GLTFSDK/Inc/GLTFSDK/ExtensionHandlers.h

```diff
@@ -183,13 +183,13 @@ namespace Microsoft
 
         class Document;
 
-        class ExtensionSerializer : public ExtensionHandlers<std::string, Extension, Document>
+        class ExtensionSerializer : public ExtensionHandlers<std::string, Extension, Document, ExtensionSerializer>
```

Comment:
I'm thinking ExtensionSerializer and ExtensionDeserializer should be made final now